include/GooseEPM/System.h

```diff
@@ -406,6 +424,24 @@ class SystemAthermal {
         return m_sigy;
     }
 
+    /**
+     * @brief Set the mean yield stress field.
```

Comment:
If we expose this I think it would make sense to also expose the standard deviation